libpod/pod.go

```diff
@@ -314,15 +314,16 @@ func (p *Pod) CgroupPath() (string, error) {
 			return "", errors.Wrapf(err, "could not get infra")
 		}
 		if ctr != nil {
-			ctr.Start(context.Background(), true)
+			// FIXME why should a call like CgroupPath() start the infra this looks wrong!!!
+			if err := ctr.Start(context.Background(), true); err != nil {
+				return "", err
+			}
 			cgroupPath, err := ctr.CgroupPath()
-			fmt.Println(cgroupPath)
```

Comment:
I guess this is some debug letfover. This leads me to believe we are not testing this code path!

Reply:
@Luap99 just looked at this and threw in some debugs. This piece of code does not seem to execute basically *ever* this is because `if p.state.CgroupPath != ""` is always true. I am going to take a look at this more in depth. Do you want to fix this in this PR? I should probably make a separate fix for this.

Reply:
If you can fix it please do. I have no understanding of the cgroup code so I would take some time for me to understand it.

Reply:
I think we can remove this entirely - using the infra container's cgroup as a parent is not safe, so this has never been called

Reply:
I will resolve this in a PR tonight @Luap99 does that sound good?